Annotato/Annotato/Views/Document/Edit/DocumentEditViewController.swift

```diff
@@ -11,7 +12,12 @@ class DocumentEditViewController: UIViewController {
 
     func initializeSubviews() {
         initializeToolbar()
-        initializeDocumentView()
+
+        if let currentDocumentViewModel = currentDocumentViewModel {
+            initializeDocumentView(documentViewModel: currentDocumentViewModel)
+        } else {
+            initializeDocumentView()
```

Comment:
Can we remove this? Seems like we shouldn't allow users to go to document view if there is no view model? But seems like you removed it in the other branch?

Reply:
Yups I removed it in another PR